client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -16,27 +16,41 @@ import { getSelectedSite } from 'state/ui/selectors';
 import Main from 'components/main';
 import Placeholder from 'my-sites/site-settings/placeholder';
 import redirectNonJetpack from 'my-sites/site-settings/redirect-non-jetpack';
+import NavigationBackButton from 'my-sites/site-settings/navigation-back-button';
 import SkipSurvey from './skip-survey';
 
 class DisconnectSite extends Component {
+	getRoute() {
+		const { siteSlug } = this.props;
+
+		if ( siteSlug ) {
```

Comment:
This is totally unnecessary with the current implementation (because we're displaying a placeholder if sites are not loaded), but just asking: is it intentional that we don't return anything if `siteSlug` is not there yet? Do we want to safeguard and return `/settings/manage-connection` there in that case?